main/settings.py

```diff
@@ -872,6 +872,16 @@ def get_all_config_keys():
 NOVOED_API_BASE_URL = get_string(
     "NOVOED_API_BASE_URL", None, description="The base URL of the NovoEd API"
 )
+NOVOED_MIT_BASE_URL = get_string(
```

Comment:
This doesn't seem to be used for anything?

Reply:
Thought I was going to need this initially. Removed 👍